### ISSUE TITLE
different weight decay between code and paper for clip 18b

### ISSUE THREAD

**Issue by Novestars** [baaivision/EVA#145]
In the paper, wd was 0, while in the code base wd is set to default value which is 0.02

**Comment by Quan-Sun** [comment]
@Novestars Appreciate you bringing this to our attention. The wd is set to 0 during the training of both EVA-CLIP-18B and EVA-CLIP-8B models.
